CADD.pm

```diff
@@ -210,17 +210,15 @@ sub run {
   if ($bvf->isa("Bio::EnsEMBL::Variation::VariationFeature")){
     $start = $bvf->{start};
     $end = $bvf->{end};
-    $allele = $bvf->alt_alleles->[$ALT_NUM];
     $ref = $bvf->ref_allele_string;
-    
-    if (($ALT_NUM + 1) >= scalar(@{$bvf->alt_alleles})) {
-      $ALT_NUM = 0;
-    } else {
-      $ALT_NUM += 1;
-    };
-    
-    return {} unless defined($allele) && $allele =~ /^[ACGT-]+$/;
-    $ALT_NUM = 0; # setting $ALT_NUM = 0 
+
+ 
+    for my $ALT_NUM (0 .. scalar(@{$bvf->alt_alleles}) - 1) {
```

Comment:
Testing `chr1    83961   rs1191138855    GAAA    AAAA,G` with and without pick  gives different results before and after adding the for loop

Reply:
Hi @likhitha-surapaneni, this has been updated as discussed, let me know if there are any issues. Thank you